certora/specs/NameWrapper.spec

```diff
@@ -64,6 +64,7 @@ definition CANNOT_CREATE_SUBDOMAIN() returns uint32 = 32;
 definition PARENT_CANNOT_CONTROL() returns uint32 = 2^16;
 definition IS_DOT_ETH() returns uint32 = 2^17;
 
+definition GRACE_PERIOD() returns uint64 = 7776000;
```

Comment:
Maybe add a comment similar to `// 90 days = 90 * 24 * 60 * 60 sec`
The original definition is in days so it might not be clear immediately what this number is

Reply:
Agreed.